### PR TITLE
[16.0][MIG] sale_order_product_recommendation: Migration to version 16.0

### PR DESCRIPTION
**Pull request by Yadier-Tecnativa**
@Tecnativa TT42432

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The width of the one2many is not fitting the whole width.

**Comment by Yadier-Tecnativa** [comment]
> The width of the one2many is not fitting the whole width.

changed

**Comment by pedrobaeza** [comment]
OK, the CI error is on other module and doesn't seem related to this.

File: sale_order_product_recommendation/wizards/sale_order_recommendation.py [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+        price_origin = (
+            fields.first(self).wizard_id.sale_recommendation_price_origin or "pricelist"
+        )
+        for one in self:
+            if price_origin == "pricelist":
+                one.price_unit = one.product_id.with_context(
+                    partner=one.partner_id.id,
+                    pricelist=one.pricelist_id.id,
+                    quantity=one.units_included,
+                ).lst_price
```
Comment (sergio-teruel) [inline sale_order_product_recommendation/wizards/sale_order_recommendation.py]: You are breaking the module behavior.
Is you get this field the sale order pricelist is omitted.
Please, use a method to get price from pricelist

You can take a look to unit_price in sale order lines

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by Yadier-Tecnativa** [comment]
There seems to be an error in the CI with the installation of the sale_partner_selectable_option module.
Is there a way not to run this module especially for this PR?

**Comment by Yadier-Tecnativa** [comment]
After many attempts to change the installation modules failed in the CI jobs, the error remains the same:
```ERROR: statement cancel due to lock timeout```
It seems something with the database implementation.

ping @pedrobaeza

**Comment by sergio-teruel** [comment]
ping @chienandalu

**Comment by sergio-teruel** [comment]
Ready to be merged!!

**Review by chienandalu — APPROVED** [review]

**Comment by sergio-teruel** [comment]
ping @juangomezzurita

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 exclude: |
   (?x)
   # NOT INSTALLABLE ADDONS
+  ^sale_automatic_workflow/|
```
Comment (pedrobaeza) [inline .pre-commit-config.yaml]: This one should be removed, isn't it?

**Comment by sergio-teruel** [comment]
Done!! waiting to CI

**Pull request merged**